app/templates/admin.html

```diff
@@ -0,0 +1,9 @@
+<html lang="en"
+<head>
+    <title>Admin Route</title>
+</head>
+
+<body>
+    <p>Admin route</p>
+</body>
```

Comment:
@hlopezes14 Let's add chart onto this secure "admin.html" which will be served login secured route. Just `git fetch` and `git merge` latest master into your working branch